hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/artifacts/smtable/SoftwareModuleDetails.java

```diff
@@ -214,4 +141,5 @@ protected String getTabSheetId() {
     protected String getDetailsHeaderCaptionId() {
         return SPUIComponetIdProvider.TARGET_DETAILS_HEADER_LABEL_ID;
     }
+
```

Comment:
No, it is not a rule that we have consistently anywhere. But nevertheless we should avoid these empty lines, shouldn't we?